src/containers/Arrivals/NewArrivals.tsx

```diff
@@ -1,73 +1,72 @@
-import ProductCard from "../../components/Products/ProductCard";
-import { useState, useEffect } from "react";
-import { useGetProductsQuery } from "../../services/products/productApi";
-import { BiSolidCircle } from "react-icons/bi";
-import { ProductResponse, Product } from "../../types/Types";
-
+import ProductCard from '../../components/Products/ProductCard';
+import { useState, useEffect } from 'react';
+import { useGetProductsQuery } from '../../services/productApi';
+import { BiSolidCircle } from 'react-icons/bi';
+import { ProductResponse, Product } from '../../types/Types';
 
 const perPage = 6;
 
 export default function NewArrivals() {
   const { data, error, isLoading } = useGetProductsQuery();
   const [currentPage, setCurrentPage] = useState(0);
 
-  console.log("helllooooo", data)
+  console.log('helllooooo', data);
 
   useEffect(() => {
-    console.log("Fetching data: isLoading?", isLoading, "Error:", error, "Products:", data);
+    console.log('Fetching data: isLoading?', isLoading, 'Error:', error, 'Products:', data);
   }, [isLoading, error, data]);
 
   if (isLoading) {
-    console.log("Loading...");
+    console.log('Loading...');
     return null;
   }
 
   if (error) {
-    console.log("Error loading products:", error);
+    console.log('Error loading products:', error);
     // Optionally display an error indicator
     return null;
   }
 
   const productsData: ProductResponse = data as unknown as ProductResponse;
 
   if (!productsData) {
-    console.log("No products data received");
+    console.log('No products data received');
     return null;
   }
```

Comment:
These codes too should be removed or replace with a proper component to handle that